compiler/tests/generated/exports/service_comm.h

```diff
@@ -1162,4 +1162,164 @@ namespace tests
     };
     
 
+    class Bar
```

Comment:
Moved inheritance test cases to `service_inheritance.bond` and run only syntax/AST tests, not codegen test since there is no codegen support for services with base yet.